package.nls.json

```
@@ -20,15 +20,18 @@
     "python.command.python.datascience.exportAsPythonScript.title": "Export as Python Script",
     "python.command.python.datascience.exportToHTML.title": "Export to HTML",
     "python.command.python.datascience.exportToPDF.title": "Export to PDF",
-    "DataScience.checkingIfImportIsSupported" : "Checking if import is supported",
-    "DataScience.installingMissingDependencies" : "Installing missing dependencies",
-    "DataScience.exportNotebookToPython" : "Exporting Notebook to Python",
-	"DataScience.performingExport" : "Performing export",
-    "DataScience.exportHTMLQuickPickLabel" : " HTML",
-	"DataScience.openExportedFileMessage" : "Would you like to open the exported file?",
-	"DataScience.openExportFileYes" : "Yes",
-	"DataScience.openExportFileNo" : "No",
-	"DataScience.failedExportMessage": "Export failed.",
+    "DataScience.checkingIfImportIsSupported": "Checking if import is supported",
+    "DataScience.installingMissingDependencies": "Installing missing dependencies",
+    "DataScience.exportNotebookToPython": "Exporting Notebook to Python",
+    "DataScience.performingExport": "Performing export",
+    "DataScience.convertingToPDF": "Converting to PDF",
+    "DataScience.exportHTMLQuickPickLabel": "HTML",
+    "DataScience.exportPDFQuickPickLabel": "PDF",
+	"DataScience.openExportedFileMessage": "Would you like to open the exported file?",
+	"DataScience.openExportFileYes": "Yes",
+    "DataScience.openExportFileNo": "No",
+    "DataScience.failedExportMessage": "Export failed.",
+    "DataScience.markdownExportToPDFDependencyMessage": "Exporting to PDF was unsuccessful. If you have not installed xelatex (TeX) you will need to do so before you can export to PDF, for further instructions please look [here](https://nbconvert.readthedocs.io/en/latest/install.html#installing-tex). \r\nTo avoid installing xelatex (TeX) you might want to try exporting to HTML and using your browsers \"Print to PDF\" feature.",
```

```suggestion
    "DataScience.exportToPDFDependencyMessage": "Exporting to PDF was unsuccessful. If you have not installed xelatex (TeX) you will need to do so before you can export to PDF, for further instructions please look [here](https://nbconvert.readthedocs.io/en/latest/install.html#installing-tex). \r\nTo avoid installing xelatex (TeX) you might want to try exporting to HTML and using your browsers \"Print to PDF\" feature.",
```

I woudldn't include the name `markdown` in  the name of the property. Tomorrow if we didn't want a link, then we need to change this property as well. How/what the format is, is not relevant.